pkg/nsxt/config/config_ini_legacy.go

```diff
@@ -28,6 +28,10 @@ func (cfg *NsxtConfigINI) ValidateConfig() error {
 		if cfg.Password == "" {
 			return errors.New("password is empty")
 		}
+	} else if cfg.ClientAuthKeyFile != "" {
```

Comment:
Do you think we also need to check the reverse where the client cert file is provided but not the key file?

Reply:
yeah agree, I feel we should also add this test case in the unit test